### PR TITLE
Add NanoAOD v7 dataset list

### PR DESCRIPTION
**Pull request by AndreasAlbert**
@alpakpinar please check

### CODE REVIEW

**Comment by AndreasAlbert** [comment]
(small fix: I was previously missing the 2018 ST_s data sets)

File: bucoffea/data/datasets/datasets_nanoaod_v7_2018.txt [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIAutumn18NanoAODv7-Nano02Apr2020_102X_upgrade2018_realistic_v21_ext1-v1/NANOAODSIM
+/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIAutumn18NanoAODv7-Nano02Apr2020_102X_upgrade2018_realistic_v21_ext2-v1/NANOAODSIM
+/TTJets_TuneCP5_13TeV-madgraphMLM-pythia8/RunIIAutumn18NanoAODv7-Nano02Apr2020_102X_upgrade2018_realistic_v21-v1/NANOAODSIM
+
+# ST
+# ST_s-channel_4f_leptonDecays_mtop1715_Tune*_PSweights_13TeV-amcatnlo-pythia8
+
+/ST_s-channel_antitop_leptonDecays_13TeV-PSweights_powheg-pythia/RunIIAutumn18NanoAODv7-Nano02Apr2020_102X_upgrade2018_realistic_v21-v1/NANOAODSIM
+/ST_s-channel_top_leptonDecays_13TeV-PSweights_powheg-pythia/RunIIAutumn18NanoAODv7-Nano02Apr2020_102X_upgrade2018_realistic_v21-v1/NANOAODSIM
+/ST_s-channel_4f_hadronicDecays_TuneCP5_13TeV-madgraph-pythia8/RunIIAutumn18NanoAODv7-Nano02Apr2020_102X_upgrade2018_realistic_v21_ext1-v1/NANOAODSIM
```
Comment (alpakpinar) [inline bucoffea/data/datasets/datasets_nanoaod_v7_2018.txt:76]: Hey @AndreasAlbert , just wanted to double check about the ST 2018 samples. In "top samples" tab in [1], I see IC is using "ST_s-channel_4f_leptonDecays_TuneCP5_13TeV-amcatnlo-pythia8". The "leptonDecays" sample we have here is named slightly differently I believe: "ST_s-channel_top_leptonDecays_13TeV-PSweights_powheg-pythia". Should we change to the other sample instead (I think I couldn't find it but sorry if I missed it). The "hadronicDecays" matches with what IC is currently using.

Thanks a lot,
Alp

[1]: https://docs.google.com/spreadsheets/d/1CkF9QeVuFUC5nZ28VvXZKFhhUNqe9sfatNOiLRgekZ4/edit#gid=312476682

Comment (alpakpinar) [reply bucoffea/data/datasets/datasets_nanoaod_v7_2018.txt:76]: I think they're also not using the "antitop" one either (L73), the one just above the dataset name I mentioned in the previous comment.

**Comment by AndreasAlbert** [comment]
Is that better? It's ok if we have extra samples in the list here, we can always reject them in the merging process later on

**Comment by alpakpinar** [comment]
I think this looks good, thank you for the update!

**Pull request merged**